sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageStreamSource.java

```diff
@@ -165,8 +165,13 @@ public String toString() {
     private GenericRecord record;
     private T current;
     private long currentOffset;
+
+    // Values used for progress reporting.
     private double fractionConsumed;
-    private double fractionConsumedFromLastResponse;
+    private double fractionConsumedFromPreviousResponse;
+    private double fractionConsumedFromCurrentResponse;
+    private long rowsReadFromCurrentResponse;
```

Comment:
I purposefully chose long variable names to avoid additional comments.

If you don't feel that the names are sufficiently self-describing, I'm happy to add more comments. Please let me know!